shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/antlr4/imports/sqlserver/SQLServerKeyword.g4

```diff
@@ -1070,3 +1066,72 @@ FILTER_COLUMN
 RETENTION_PERIOD
     : R E T E N T I O N UL_ P E R I O D
     ;
+
+CONTAINMENT
+    : C O N T A I N M E N T
+    ;
+
+PARTIAL
+    : P A R T I A L
+    ;
+
+FILENAME
+    : F I L E N A M E
+    ;
+
+SIZE
+    : S I Z E
+    ;
+
+MAXSIZE
+    : M A X S I Z E
+    ;
+
+FILEGROWTH
+    : F I L E G R O W T H
+    ;
+
+UNLIMITED
+    : U N L I M I T E D
+    ;
+
+KB
+    : K B
+    ;
+
+MB
+    : M B
+    ;
+
+GB
+    : G B
+    ;
+
+TB
+    : T B
+    ;
+
+CONTAINS
+    : C O N T A I N S
+    ;
+
+MEMORY_OPTIMIZED_DATA
+    : M E M O R Y UL_ O P T I M I Z E D UL_ D A T A
+    ;
+
+FILEGROUP
+    : F I L E G R O U P
+    ;
+
+NON_TRANSACTED_ACCESS
+    : N O N UL_ T R A N S A C T E D UL_ A C C E S S
+    ;
+
+DB_CHAINING
+    : D B UL_ C H A I N I N G
+    ;
+
+TRUSTWORTHY
+    : T R U S T W O R T H Y
+    ;
+
```

Comment:
@Icesource  If these tokens is not reserved keyword, maybe these should be contained in `unreservedWord `(BaseRule.g4).

Reply:
@jingshanglu done